dnf-docker-test/features/module-install-pkg-3.feature

```diff
@@ -27,3 +28,26 @@ Feature: Installing package when default stream is defined
           | version | -1    |
           | enabled | 1     |
           | stream  | f26   |
+
+  Scenario: cleanup from previous scenario
+      Given I successfully run "dnf remove TestY -y"
+        And I successfully run "dnf module disable ModuleY"
+
+  Scenario: rpm from enabled stream is preferred regardless of NVRs
```

Comment:
Given that you have default stream still defined how is this scenario different from the previous one?
I think you should rather enable f27 (non-default) stream here.